api/build.gradle

```diff
@@ -225,9 +223,11 @@ swaggerSources {
 //
 
 // Generate AppEngine Cron file
+// Use local env if WORKBENCH_ENV not set.
+def workbenchEnv = System.getenv('WORKBENCH_ENV') != null ? System.getenv('WORKBENCH_ENV') : 'local'
```

Comment:
You might prefer to require it rather than set a default. You could:
```groovy
def workbenchEnv = java.util.Objects.requireNonNull(System.getenv('WORKBENCH_ENV'), 'WORKBENCH_ENV must be defined.')
```

Reply:
Just checked with @calbach seems unit test does not set `WORKBENCH_ENV`, so it is expected to get null here for local run. Let me check where we can set that for local run/unit test scenario.